index.html

```diff
@@ -5510,14 +5524,65 @@ <h2 id="instantiation-of-AudioWorkletNode-and-AudioWorkletProcessor">
             <li>Set <a>processor reference</a> in <var>this</var> to
             <var>processorInstance</var>.
             </li>
-            <li>Add <var>nodeInstance</var> to the <a>node instance set</a> in
-            the <a>BaseAudioContext</a>.
-            </li>
             <li>Return <var>this</var> as a result of the construction.
             </li>
           </ol>
         </section>
         <section class="informative">
+          <h2 id="AudioWorklet-Sequence">
+            AudioWorklet Sequence of Events
+          </h2>
+          <p>
+            The following figure illustrates an idealized sequence of events
+            occurring relative to an <a>AudioWorklet</a>:
+          </p>
+          <figure>
+            <img alt="AudioWorklet sequence" src=
+            "images/audioProcessorInstantiation.png" width="917" height="546">
```

Comment:
Agreed. We land the diagram as it is and can fix it later if necessary. When we freeze the spec for CR, I would like to rework on every diagram in the spec.